### ISSUE TITLE
resin build fails on Windows 10

### ISSUE THREAD

**Issue by emilm** [balena-io/balena-cli#961]
- **resin-cli version:**
7.9.3
- **Operating system and architecture:**
Windows 10 pro, x64

I try to build something that works on your servers.

I get: _COPY failed: stat /var/lib/docker/tmp/docker-builder233937031/myapp/pom.xml: no such file or directory_

On another project I get: _myapp2 [===> ] 15% Step 4/26: Unknown host QEMU_IFLA type: 41_

When I look inside the image, /var/lib/docker does not exist. Support also thinks the context is not transferred.

Normal docker build works. Hope you understand, I do not want to publish any docker files or anything here because that would break NDA. :)

<img src="https://frontapp.com/assets/img/icons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_dk1f)

**Comment by CameronDiver** [comment]
@emilm I don't have much experience with Windows + Docker, but I'd imagine it'd be difference in setup in certain circumstances. Is there any other information you could provide us with about your projects or Docker setup?

Things like;
* Docker version
* Your build command
* Can you build very simple Dockerfiles, for example:
```Dockerfile
FROM debian
WORKDIR /usr/src/app
COPY Dockerfile .
CMD while true; do echo "hello"; sleep 5; done;
```

**Comment by pdcastro** [comment]
I believe this issue was fixed in CLI v11 (currently v11.18.2) -
```
C:\Users\paulo>balena build C:\Users\paulo\balena\other\simple_container --nocache --logs -a test-rpi
[Info]    Creating default composition with source: C:\Users\paulo\balena\other\simple_container
[Info]    Building for armv7hf/raspberrypi3
[Build]   Building services...
[Build]   main Preparing...
[Build]   main Step 1/3 : FROM arm32v7/busybox
[Build]   main  ---> 2dcbd0d94d0b
[Build]   main Step 2/3 : RUN uname -a
[Build]   main  ---> Running in a307cb33e2fa
[Build]   main Linux a307cb33e2fa 4.9.93-linuxkit-aufs #1 SMP Wed Jun 6 16:55:56 UTC 2018 armv7l GNU/Linux
[Build]   main Removing intermediate container a307cb33e2fa
[Build]   main  ---> ced19a265141
[Build]   main Step 3/3 : CMD while : ; do echo "(Plain Dockerfile 30) $(uname -a)"; sleep ${INTERVAL=5}; done
[Build]   main  ---> Running in 9bde3d24ca2b
[Build]   main Removing intermediate container 9bde3d24ca2b
[Build]   main  ---> 0b55ef6f8533
[Build]   main Successfully built 0b55ef6f8533
[Build]   main Successfully tagged simple_container_main:latest
[Build]   main Waiting...
[Build]   Built 1 service in 0:06
[Success] Build succeeded!

C:\Users\paulo>balena version -a
balena-cli version "11.18.2"
Node.js version "8.15.1"

C:\Users\paulo>docker version
Client:
 Version:           18.06.1-ce
 API version:       1.38
 Go version:        go1.10.3
 Git commit:        e68fc7a
 Built:             Tue Aug 21 17:21:34 2018
 OS/Arch:           windows/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.06.1-ce
  API version:      1.38 (minimum version 1.12)
  Go version:       go1.10.3
  Git commit:       e68fc7a
  Built:            Tue Aug 21 17:29:02 2018
  OS/Arch:          linux/amd64
  Experimental:     false
```

I tested it with a couple of simple Dockerfiles, including the one suggested by Cameron:
```
FROM arm32v7/busybox
RUN uname -a
CMD while : ; do echo "(Plain Dockerfile 30) $(uname -a)"; sleep ${INTERVAL=5}; done
```

```
FROM debian
WORKDIR /usr/src/app
COPY Dockerfile .
CMD while true; do echo "hello"; sleep 5; done;
```
